ufo2fontir/src/source.rs

```diff
@@ -836,7 +858,11 @@ impl Work<Context, WorkError> for FeatureWork {
         }
 
         if !fea_files.is_empty() {
-            context.set_features(Features::from_file(&fea_files[0]));
+            let designspace_dir = self.designspace_file.parent().unwrap();
+            context.set_features(Features::from_file(
+                fea_files[0].clone(),
+                Some(designspace_dir.to_path_buf()),
```

Comment:
no, it's not "relative to the directory of the .designspace", it's relative to the directory of the UFO. The designspace spec doesn't say anything about features.fea, it's the UFO spec that mandates that:
https://unifiedfontobject.org/versions/ufo3/features.fea/

it happens to work with FirerSans because in that particular case the source UFOs which contain features.fea with include statements happen to be in the same directory as the .designspace file that glues them together, but they may well be in a different sub-directory

Reply:
Note to self: directory of the UFO means the directory containing it, not the actual blah.ufo/ directory

Reply:
I will modify my test files to expose the bug & fix.